dist/gcc-compatible/Hacl_Hash_SHA2.h

```diff
@@ -77,13 +77,13 @@ Hacl_Hash_SHA2_update_last_512(
   uint32_t input_len
 );
 
-void Hacl_Hash_SHA2_hash_224(uint8_t *input, uint32_t input_len, uint8_t *dst);
+void Hacl_Hash_SHA2_hash_224(uint8_t *output, uint8_t *input, uint32_t input_len);
```

Comment:
Al the SHA/MD functions are lacking comments. But we can do this in a follow up.